e2e/_suites/fleet/fleet.go

```diff
@@ -776,6 +776,13 @@ func bootstrapFleet(ctx context.Context, env map[string]string) error {
 				Flavour: "fleet-server",
 			}
 
+			// we have detected that fleet-server immediately dies when started and kibana is slowest in provide the initial requirements for fleet-server
+			// as we do not know what is the right healtch check to apply here, we are sleeping the execution 1 minute because:
+			//   1. it takes 10 minutes to fail because if that
+			//   2. it takes ~3 minutes more to sucessfully start the elastic stack (using our retry at the CI level)
+			//   3. we expect this minute saves ~5
```

Comment:
what does it mean in this context? 5 times or  5 hours?

Reply:
5 minutes